crates/gpio/src/backend.rs

```diff
@@ -237,18 +242,14 @@ fn start_backend<D: 'static + GpioDevice + Send + Sync>(args: GpioArgs) -> Resul
 pub(crate) fn gpio_init() {
     env_logger::init();
 
-    if let Err(e) = start_backend::<PhysDevice>(GpioArgs::parse()) {
-        error!("{e}");
-        exit(1);
-    }
+    let _ = start_backend(GpioArgs::parse());
```

Comment:
Why don't we need this exit and error thing anymore ?

Reply:
Yeah we do - we can fail at the config and the last commit even allows failures from the spawn to propagate out so I'll restore the exit case.